sonar-csharp-plugin/src/main/java/org/sonar/plugins/csharp/CSharp.java

```diff
@@ -36,4 +36,8 @@ public CSharp(Settings settings) {
     return settings.getStringArray(CSharpPlugin.FILE_SUFFIXES_KEY);
   }
 
+  @Override
+  public boolean equals(Object o) {
```

Comment:
Unfinished job sorry, wanted to fix the warning and didn't finished the implementation before getting disrupted.